ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -3086,6 +3086,13 @@ Seems much cleaner if each stmt is identified as a particular HiveOperation (whi
             output.getWriteType().name()));
         break;
 
+      case CTAS:
+        if (AcidUtils.isExclusiveCTASEnabled(t, conf)) {
+          compBuilder.setExclWrite();
+          compBuilder.setOperationType(DataOperationType.NO_TXN);
+          break;
+        }
```

Comment:
should we continue otherwise?

Reply:
Yes, it should continue if it is not a transactional table or the config is not enabled.
This was the old behavior as well.

Reply:
i do not see `continue` keyword